packages/core-database-postgres/lib/connection.js

```diff
@@ -185,7 +183,10 @@ module.exports = class PostgresConnection extends ConnectionInterface {
       currentSeed = crypto.createHash('sha256').update(currentSeed).digest()
     }
 
-    this.activeDelegates = data
+    this.activeDelegates = data.map(delegate => {
+      delegate.round = +round
```

Comment:
what is the rationale for this?

Reply:
There is a comparison somewhere that was always `false` because it compared `string === integer`, this fixes the comparison to `integer === integer`.

Reply:
i mean why adding delegate.round property?

Reply:
ah you mean you override the round property of delegates?

Reply:
@supaiku0 added it for here https://github.com/ArkEcosystem/core/blob/develop/packages/core-database-postgres/lib/connection.js#L167

Reply:
@fix See https://github.com/ArkEcosystem/core/pull/1143 and https://github.com/ArkEcosystem/core/pull/1152